policy/test/inputs/psa-endorsements.json

```diff
@@ -0,0 +1,2 @@
+
+["{\n\"scheme\":\"PSA_IOT\",\n\"type\":\"REFERENCE_VALUE\",\n\"attributes\":{\n \"psa.hw-model\":\"RoadRunner\",\n \"psa.hw-vendor\":\"ACME\",\n  \"psa.impl-id\":\"76543210fedcba9817161514131211101f1e1d1c1b1a1918\",\n    \"psa.measurement-desc\":1,\n    \"psa.measurement-type\":\"BL\",\n    \"psa.measurement-value\":\"BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=\",\n    \"psa.signer-id\":\"BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=\",\n    \"psa.version\":\"3.4.2\"\n    }\n  }","\n{\n    \"scheme\":\"PSA_IOT\",\n    \"type\":\"REFERENCE_VALUE\",\n    \"attributes\":{\n        \"psa.hw-model\":\"RoadRunner\",\n        \"psa.hw-vendor\":\"ACME\",\n        \"psa.impl-id\":\"76543210fedcba9817161514131211101f1e1d1c1b1a1918\",\n        \"psa.measurement-desc\":1,\n        \"psa.measurement-type\":\"M1\",\n        \"psa.measurement-value\":\"CwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=\",\n        \"psa.signer-id\":\"BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=\",\n        \"psa.version\":\"1.2.0\"}\n  }\n ","\n  {\n    \"scheme\":\"PSA_IOT\",\n    \"type\":\"REFERENCE_VALUE\",\n    \"attributes\":{\n        \"psa.hw-model\":\"RoadRunner\",\n        \"psa.hw-vendor\":\"ACME\",\n        \"psa.impl-id\":\"76543210fedcba9817161514131211101f1e1d1c1b1a1918\",\n        \"psa.measurement-desc\":1,\n        \"psa.measurement-type\":\"M2\",\n        \"psa.measurement-value\":\"DwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=\",\n        \"psa.signer-id\":\"BwYFBAMCAQAPDg0MCwoJCBcWFRQTEhEQHx4dHBsaGRg=\",\n        \"psa.version\":\"1.2.3\"}\n  }\n  "]
```

Comment:
not sure, why is this encoded this way?

Reply:
This decodes to an array of stings, which is what is returned as endorsements from the kvstore.